src/Mono.Android/Android.App/UsesConfigurationAttribute.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+using Android.Content.PM;
+using Android.Views;
+
+namespace Android.App {
+
+	[Serializable]
+	[AttributeUsage (AttributeTargets.Assembly | AttributeTargets.Class, 
```

Comment:
What does it mean for this attribute to be placed on `AttributeTargets.Class`? If it could be placed within an `<activity/>` that would make sense, but it can't, as far as I can tell...

Reply:
Good call, it should not happen. Fixing.